### PR TITLE
Change lexical scope motors, sensors, configuration

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

File: src/js/tabs/configuration.js [hunk 1]
```diff
@@ -968,14 +968,14 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
...
                     'Onboard ADC',
-                    'ESC Sensor'
+                    'ESC Sensor',
                 ];
 
-                var batteryMeterType_e = $('select.batterymetertype');
-                for (i = 0; i < batteryMeterTypes.length; i++) {
-                    batteryMeterType_e.append('<option value="' + i + '">' + batteryMeterTypes[i] + '</option>');
+                const batteryMeterType_e = $('select.batterymetertype');
+                for (const index in batteryMeterTypes) {
```
Comment (mikeller) [inline src/js/tabs/configuration.js:977]: `for ( of )` or leave as is.

File: src/js/tabs/configuration.js [hunk 2]
```diff
@@ -999,19 +999,19 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
...
 
             if (semver.gte(FC.CONFIG.flightControllerVersion, "3.1.0")) {
                 currentMeterTypes.push('ESC Sensor');
             }
 
-            var currentMeterType_e = $('select.currentmetertype');
-            for (i = 0; i < currentMeterTypes.length; i++) {
-                currentMeterType_e.append('<option value="' + i + '">' + currentMeterTypes[i] + '</option>');
+            const currentMeterType_e = $('select.currentmetertype');
+            for (const index in currentMeterTypes) {
```
Comment (mikeller) [inline src/js/tabs/configuration.js:1013]: `for ( of )` or leave as is.

File: src/js/tabs/motors.js [hunk 3]
```diff
@@ -126,14 +126,14 @@ TABS.motors.initialize = function (callback) {
             }
         }
         while (data[0].length > 300) {
-            for (i = 0; i < data.length; i++) {
-                data[i].shift();
+            for (const index in data) {
```
Comment (mikeller) [inline src/js/tabs/motors.js:129]: `for ( of )` or leave as is.

**Review by mikeller — CHANGES_REQUESTED** [review]

**Comment by haslinghuis** [comment]
Thanks. Fixed as requested per review.

File: src/js/tabs/motors.js [hunk 4]
```diff
@@ -126,14 +126,14 @@ TABS.motors.initialize = function (callback) {
             }
         }
         while (data[0].length > 300) {
-            for (i = 0; i < data.length; i++) {
-                data[i].shift();
+            for (let i = 0; i < data.length; i++) {
+                index.shift();
```
Comment (mikeller) [inline src/js/tabs/motors.js:130]: Where's this `index` from?

Please at least check your code for errors before pushing.

**Review by mikeller — CHANGES_REQUESTED** [review]

File: src/js/tabs/motors.js (hunk 4 above)
Comment (mikeller) [inline src/js/tabs/motors.js:130]: Where's this index from?

Please at least check your code for errors before pushing.

**Review by mikeller — CHANGES_REQUESTED** [review]

File: src/js/tabs/motors.js (hunk 4 above)
Comment (haslinghuis) [reply src/js/tabs/motors.js:130]: Oops this was the only was I didn't test after changes. Sorry. GUI was working, didn't look at the console :-( 
Fixed.

**Review by mikeller — APPROVED** [review]

**Pull request merged**